### ISSUE TITLE
Index-worthy projects

### ISSUE THREAD

**Issue by mosteo** [alire-project/alire#193]
Umbrella issue for projects that may be worth to index if their authors don't step up.

- [ ] LibAdaLang
- [ ] EE KDF9 emulator
- [ ] Gautier's projects
- [ ] https://github.com/LionelDraghi/List_Image
- [ ] jrcarter/PragmARC

**Comment by Patschkowski** [comment]
Do you consider AZip a duplicate with Gautier's project?

**Comment by mosteo** [comment]
You're right, that's one of him. No matter, this is more of a laundry list than a strictly prioritized to-do. And, hopefully, authors' themselves will contribute their projects at some point if alire gets enough traction.

**Comment by onox** [comment]
aws, libadalang, gwindows, and win32ada seem to have been added to the index.
